### ISSUE TITLE
[Feature Request] Add support of both `vips_composite` AND `vips_composite2`

### ISSUE THREAD

**Issue by zzjin** [davidbyttow/govips#129]
I love the func `Composite` the govips provided.
But in my case, I need composite multiple images(>10) together.
When dig into sources I found go func `Composite` call `vips_composite2` inside.
Maybe add func `Composite` to inner call `vips_composite` and change  current `Composite` to `Composite2` will better match original libvips behavior.

Although multiple call `Composite` inside loop will get same result, but performance maybe diff.

PS: I'm working on one PR but I'm not familiar cgo, and something hard to call `C.vips_composite`

```go
// https://libvips.github.io/libvips/API/current/libvips-conversion.html#vips-composite
func vipsComposite(ins []*C.VipsImage, modes []BlendMode, x, y []int) (*C.VipsImage, error) {
	incOpCounter("composite")
	var out *C.VipsImage

	//TODO: convert go slice to c *int

	if err := C.composite_image(&ins[0], &out, C.int(len(ins)), &mode, &x, &y); err != 0 {
		return nil, handleImageError(out)
	}

	return out, nil
}
```

**Comment by tonimelisma** [comment]
Hey @zzjin! Thanks for being active. We want to maintain backwards compatibility so I don't think it's a good idea to change the behavior of existing functions. Adding the other function is more than welcome, though. I await a PR! 👍 

I feel you, cgo is quite tricky to write and there isn't much great documentation on the topic. Best I can refer you is to [https://golang.org/cmd/cgo/#hdr-Passing_pointers](https://golang.org/cmd/cgo/#hdr-Passing_pointers). You're looking for C.int but pay attention to the advice on passing pointers.

**Comment by tonimelisma** [comment]
Closed by #132 - great work, thank you for the contribution!
